geo/src/algorithm/mod.rs

```diff
@@ -1,93 +1,188 @@
 /// Kernels to compute various predicates
 pub mod kernels;
+pub use kernels::{HasKernel, Kernel, Orientation};
 
 /// Calculate the area of the surface of a `Geometry`.
 pub mod area;
+pub use area::Area;
+
 /// Calculate the bearing to another `Point`, in degrees.
 pub mod bearing;
+pub use bearing::Bearing;
+
 /// Calculate the bounding rectangle of a `Geometry`.
 pub mod bounding_rect;
+pub use bounding_rect::BoundingRect;
+
 /// Calculate the centroid of a `Geometry`.
 pub mod centroid;
+pub use centroid::Centroid;
+
 /// Smoothen `LineString`, `Polygon`, `MultiLineString` and `MultiPolygon` using Chaikins algorithm.
 pub mod chaikin_smoothing;
+pub use chaikin_smoothing::ChaikinSmoothing;
+
 /// Calculate the signed approximate geodesic area of a `Geometry`.
 pub mod chamberlain_duquette_area;
+pub use chamberlain_duquette_area::ChamberlainDuquetteArea;
+
 /// Calculate the closest `Point` between a `Geometry` and an input `Point`.
 pub mod closest_point;
+pub use closest_point::ClosestPoint;
+
 /// Calculate the concave hull of a `Geometry`.
 pub mod concave_hull;
+pub use concave_hull::ConcaveHull;
+
 /// Determine whether `Geometry` `A` is completely enclosed by `Geometry` `B`.
 pub mod contains;
+pub use contains::Contains;
+
 /// Calculate the convex hull of a `Geometry`.
 pub mod convex_hull;
+// REVIEW: use qhull, etc?
```

Comment:
My vote is 👎🏻 for all the `REVIEW` comments in this file

Reply:
Removed in https://github.com/georust/geo/pull/830/commits/700f008ed4b50040093b13d4cfa2aa7c6bd0aeb8